### ISSUE TITLE
Update docs at reposilite-site to 3.x

### ISSUE THREAD

**Issue by dzikoysk** [dzikoysk/reposilite#557]

**Comment by EthanDevelops** [comment]
Maybe create a way to view docs for different versions?

**Comment by dzikoysk** [comment]
Yeah, that's already resolved in #994, but the current website still misses updated content, so it's not mentioned directly.

**Comment by fklappan** [comment]
Maybe we should gather some requirements which is needed for the 3.x docs

**Comment by dzikoysk** [comment]
I need to find a day I can reorganize all docs pages by some kind of a groups that have much more sense that it is right now. Speaking of content, there are some aspects of 3.x that are still open to breaking changes, so I think we need to wait for stable release to be sure docs are up-to-date.
